reference-data/identifier-types/isbn.json

```diff
@@ -0,0 +1,5 @@
+{
+    "id":"8261054f-be78-422d-bd51-4ed9f33c3422",
+    "name":"isbn",
+    "source": "folio"
+}
```

Comment:
As far as I know, circulation-storage does not provide a set of `identifier types`. What is this used for?

Reply:
The model I was following for this work had a similar file-I assumed it was necessary for associating this specific UUID with isbn identifiers, so that they could be retrieved by that UUID later (for example in tests, which pull the identifiers based on the UUID provided here).  When I examined the isbn identifiers in the tests, they had this UUID for isbn identifier type, which seemed to confirm that theory.

Reply:
> I assumed it was necessary for associating this specific UUID with isbn identifiers

I don't think it is needed at all.

> for example in tests, which pull the identifiers based on the UUID provided here

How is that done? As far as I can tell, there is no API for identifier types in mod-circulation-storage and no code to load this record.

> When I examined the isbn identifiers in the tests, they had this UUID for isbn identifier type, which seemed to confirm that theory.

I imagine that is because it is hardcoded in the test records.